assets/rich_text_components_definitions.ts

```diff
@@ -102,8 +102,12 @@ export default {
       "name": "url",
       "description": "The link URL. If no protocol is specified, HTTPS will be used.",
       "schema": {
-        "type": "custom",
-        "obj_type": "SanitizedUrl"
+        "type": "unicode",
+        "obj_type": "SanitizedUrl",
```

Comment:
Let's try to keep the `schema type` for `Link`-s to be "custom". Since this is a custom schema, it actually has its own file (`extensions/objects/templates/sanitized-url-editor.component.ts`). Modify the `validators` portion of `SanitizedUrlSchema` to be like what you have below, and also modify `interface SanitizedUrlSchema`!

Reply:
@EricZLou thank you for your tips, I pushed another commit with the changes suggested by you.